samples/csharp/end-to-end-apps/ObjectDetection-Onnx/OnnxObjectDetectionApp/MainWindow.xaml.cs

```diff
@@ -0,0 +1,203 @@
+﻿using Microsoft.ML;
+using OnnxObjectDetection;
+using OpenCvSharp;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Diagnostics;
+using System.Drawing;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Media;
+using System.Windows.Media.Imaging;
+using Rectangle = System.Windows.Shapes.Rectangle;
+
+namespace OnnxObjectDetectionApp
+{
+    /// <summary>
+    /// Interaction logic for MainWindow.xaml
+    /// </summary>
+    public partial class MainWindow : System.Windows.Window
+    {
+
+        //private Mat frame;
+        private VideoCapture capture;
+
+        private CancellationTokenSource cameraCaptureCancellationTokenSource;
+
+        //private readonly MLContext mlContext = new MLContext();
+        private readonly YoloOutputParser yoloParser = new YoloOutputParser();
+        private PredictionEngine<ImageInputData, ImageObjectPrediction> predictionEngine;
+
+
+        public MainWindow()
```

Comment:
MainWindow, OnActivated, OnDeactivated, LoadModel, and StartCameraCapture feel especially spaced out- could we condense it a bit/make the format more consistent by removing some of the spaces between lines? At least for the 2 line methods, it might look cleaner if there aren't spaces between each line.

Reply:
Resolved 368f84c2f15116ae98ddd8d797e326e953544dcc